tests/app/helpers.py

```diff
@@ -7,11 +7,10 @@
 
 from app import create_app
 from app import login_manager
-from unittest import TestCase
 
 
-class BaseApplicationTest(TestCase):
-    def setUp(self):
+class BaseApplicationTest(object):
+    def setup_method(self, method):
```

Comment:
Ah, okay, I thought that these things could be called just once for all of the tests in a class, but I guess it's cleaner to reset for each one. (Which kinda begs the question why one uses a class at all, if no state is allowed to be shared between the method invocations, but eh.)